dist/tools/coccinelle/check.sh

```diff
@@ -6,9 +6,8 @@
 # General Public License v2.1. See the file LICENSE in the top level
 # directory for more details.
 
-: ${RIOTBASE:=$(pwd)}
-
-. ${RIOTBASE}/dist/tools/ci/changed_files.sh
+: "${RIOTTOOLS:=./dist/tools}"
```

Comment:
shouldn't this better be `: "${RIOTTOOLS:=$(pwd)/dist/tools}"`

Reply:
Some of the files had './dist/tools' before, but putting a safe default value as example is better.
One could use a 'cd' in one command and only fail when executed without 'RIOTTOOLS' set.